tests/test_manifest.py

```diff
@@ -131,7 +132,8 @@ def test_sdist_excludes_unwanted_files(self):
 		]
 		for unwanted_file in unwanted_files:
 			self.assertNotIn(
-				unwanted_file, members,
+				unwanted_file,
+				members,
 				str("""Unwanted file {reject} found in sdist.""").format(reject=unwanted_file)
```

Comment:
This could also be improved as per #134 with F-strings.

```suggestion
			f"Unwanted file {str(unwanted_file)} found in sdist."
```